<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>892</width>
    <height>402</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>TurnFix</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../resources/TurnFix.qrc">
    <normaloff>:/appicons/icon.png</normaloff>:/appicons/icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QToolBar {
        border: 0;
        font-weight: bold;
        border-right:1px solid rgba(57, 57, 57, 255);
        spacing:0;
}
QToolBar:left {
        width:90px;
        background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(57, 57, 57, 255), stop:1 rgba(130, 130, 130, 255));
}

QToolButton {
        width:110px;
        border:0;
        padding:1px;
        border-right:1px solid rgba(57, 57, 57, 255);
        border-top:1px solid qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(57, 57, 57, 255), stop:1 rgba(130, 130, 130, 255));
        border-bottom:1px solid qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(57, 57, 57, 255), stop:1 rgba(130, 130, 130, 255));
        color:white;
        font-weight:bold;
        font-size:10px;
}

QToolButton:checked, QToolButton:selected {
    color:#000000;
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(196, 196, 196, 255), stop:1 rgba(252, 252, 252, 255));
        border-top:1px solid rgba(57, 57, 57, 255);
        border-bottom:1px solid rgba(57, 57, 57, 255);
}

QToolButton:disabled {
        color:#cccccc;
}

QWidget#sidebar {
        background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(57, 57, 57, 255), stop:1 rgba(130, 130, 130, 255));
        border-right:1px solid rgba(57, 57, 57, 255);
}

QWidget#headerWidget {
        background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(255, 131, 0, 255), stop:1 rgba(255, 223, 100, 255));
                border-bottom:1px solid rgba(57, 57, 57, 255);
}

QWidget#toolBar {
        background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(57, 57, 57, 255), stop:1 rgba(180, 180, 180, 255));
}

QPushButton {
        background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(196, 196, 196, 255), stop:1 rgba(252, 252, 252, 255));
        border:1px solid rgb(206, 206, 206);
        border-radius:5px;
        padding:2px;
}

QPushButton:pressed, QPushButton:hover {
        border: 1px solid rgb(255, 170, 0);
        background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(255, 131, 0, 255), stop:1 rgba(255, 223, 100, 255));
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>454</width>
     <height>0</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0" rowspan="2" colspan="2">
     <widget class="QWidget" name="headerWidget" native="true">
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="leftMargin">
        <number>4</number>
       </property>
       <property name="topMargin">
        <number>4</number>
       </property>
       <property name="rightMargin">
        <number>4</number>
       </property>
       <property name="bottomMargin">
        <number>4</number>
       </property>
       <property name="verticalSpacing">
        <number>0</number>
       </property>
       <item row="0" column="1" rowspan="2">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="2" rowspan="2">
        <widget class="QLabel" name="label_2">
         <property name="styleSheet">
          <string notr="true">font-size:24px;
font-weight:bold;
font-family:Tahoma;</string>
         </property>
         <property name="text">
          <string>TurnFix</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3" rowspan="2">
        <widget class="QLabel" name="label">
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="../../resources/TurnFix.qrc">:/appicons/icon.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="lbl_wk">
         <property name="font">
          <font>
           <pointsize>-1</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">font-size:13px;
font-weight:bold;</string>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lbl_ort">
         <property name="styleSheet">
          <string notr="true">font-size:10px;</string>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QStackedWidget" name="sta_wdg">
      <widget class="Tab_WK" name="wk_tab"/>
      <widget class="ParticipantsWidget" name="tn_tab"/>
      <widget class="Tab_RG" name="rg_tab"/>
      <widget class="Tab_SR" name="sr_tab"/>
      <widget class="Tab_PE" name="pe_tab"/>
      <widget class="Tab_ER" name="er_tab"/>
      <widget class="Tab_DR" name="dr_tab"/>
      <widget class="Tab_ST" name="st_tab"/>
      <widget class="Tab_ED" name="ed_tab"/>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QWidget" name="sidebar" native="true">
      <property name="minimumSize">
       <size>
        <width>110</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>110</width>
        <height>16777215</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>892</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuWettkampf">
    <property name="title">
     <string>Wettkampf</string>
    </property>
    <addaction name="act_change_contest"/>
    <addaction name="separator"/>
    <addaction name="act_preferences"/>
    <addaction name="act_nos"/>
    <addaction name="separator"/>
    <addaction name="act_mail"/>
    <addaction name="separator"/>
    <addaction name="act_exit"/>
   </widget>
   <widget class="QMenu" name="menuDatenbank">
    <property name="title">
     <string>Datenbank</string>
    </property>
    <addaction name="act_competitordatabase"/>
    <addaction name="act_clubdatabase"/>
    <addaction name="act_events"/>
    <addaction name="separator"/>
    <addaction name="act_pass"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>?</string>
    </property>
    <addaction name="act_license"/>
   </widget>
   <addaction name="menuWettkampf"/>
   <addaction name="menuDatenbank"/>
   <addaction name="menu"/>
  </widget>
  <action name="act_change_contest">
   <property name="text">
    <string>Wettkampf wechseln</string>
   </property>
  </action>
  <action name="act_preferences">
   <property name="text">
    <string>Eigenschaften</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
  <action name="act_exit">
   <property name="text">
    <string>Beenden</string>
   </property>
   <property name="menuRole">
    <enum>QAction::QuitRole</enum>
   </property>
  </action>
  <action name="act_competitordatabase">
   <property name="text">
    <string>Turnerdatenbank</string>
   </property>
   <property name="shortcut">
    <string>F3</string>
   </property>
  </action>
  <action name="act_clubdatabase">
   <property name="text">
    <string>Vereinsdatenbank</string>
   </property>
   <property name="shortcut">
    <string>F4</string>
   </property>
  </action>
  <action name="act_events">
   <property name="text">
    <string>Disziplinen</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="act_license">
   <property name="text">
    <string>Über Turnfix</string>
   </property>
  </action>
  <action name="act_nos">
   <property name="text">
    <string>Startnummern vergeben</string>
   </property>
  </action>
  <action name="act_mail">
   <property name="text">
    <string>Listen per Email verschicken</string>
   </property>
  </action>
  <action name="act_pass">
   <property name="text">
    <string>Startpassnummern erfassen</string>
   </property>
  </action>
  <action name="act_WK">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../resources/TurnFix.qrc">
     <normaloff>:/appicons/icon.png</normaloff>:/appicons/icon.png</iconset>
   </property>
   <property name="text">
    <string>Wettkämpfe</string>
   </property>
  </action>
  <action name="act_TN">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../resources/TurnFix.qrc">
     <normaloff>:/appicons/teilnehmer.png</normaloff>:/appicons/teilnehmer.png</iconset>
   </property>
   <property name="text">
    <string>Teilnehmer</string>
   </property>
  </action>
  <action name="act_RG">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../resources/TurnFix.qrc">
     <normaloff>:/appicons/riegen.png</normaloff>:/appicons/riegen.png</iconset>
   </property>
   <property name="text">
    <string>Riegen einteilen</string>
   </property>
  </action>
  <action name="act_PE">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../resources/TurnFix.qrc">
     <normaloff>:/appicons/leistungen.png</normaloff>:/appicons/leistungen.png</iconset>
   </property>
   <property name="text">
    <string>Leistungen erfassen</string>
   </property>
  </action>
  <action name="act_ER">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../resources/TurnFix.qrc">
     <normaloff>:/appicons/ergebnisse.png</normaloff>:/appicons/ergebnisse.png</iconset>
   </property>
   <property name="text">
    <string>Ergebnisse</string>
   </property>
  </action>
  <action name="act_DR">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../resources/TurnFix.qrc">
     <normaloff>:/appicons/drucken.png</normaloff>:/appicons/drucken.png</iconset>
   </property>
   <property name="text">
    <string>Drucken &amp;&amp; Exportieren</string>
   </property>
  </action>
  <action name="act_ST">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../resources/TurnFix.qrc">
     <normaloff>:/appicons/status.png</normaloff>:/appicons/status.png</iconset>
   </property>
   <property name="text">
    <string>Statusübersicht</string>
   </property>
  </action>
  <action name="act_ED">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../resources/TurnFix.qrc">
     <normaloff>:/appicons/layout.png</normaloff>:/appicons/layout.png</iconset>
   </property>
   <property name="text">
    <string>Layout-Editor</string>
   </property>
  </action>
  <action name="act_SR">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../resources/TurnFix.qrc">
     <normaloff>:/appicons/directory_listing.png</normaloff>:/appicons/directory_listing.png</iconset>
   </property>
   <property name="text">
    <string>Startreihenfolgen</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Tab_WK</class>
   <extends>QWidget</extends>
   <header>src/widgets/header/wdg_tab_wk.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ParticipantsWidget</class>
   <extends>QWidget</extends>
   <header>participants/participantswidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Tab_RG</class>
   <extends>QWidget</extends>
   <header>src/widgets/header/wdg_tab_rg.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Tab_PE</class>
   <extends>QWidget</extends>
   <header>src/widgets/header/wdg_tab_pe.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Tab_ER</class>
   <extends>QWidget</extends>
   <header>src/widgets/header/wdg_tab_er.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Tab_DR</class>
   <extends>QWidget</extends>
   <header>src/widgets/header/wdg_tab_dr.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Tab_ST</class>
   <extends>QWidget</extends>
   <header>src/widgets/header/wdg_tab_st.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Tab_ED</class>
   <extends>QWidget</extends>
   <header>src/widgets/header/wdg_tab_ed.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Tab_SR</class>
   <extends>QWidget</extends>
   <header>src/widgets/header/wdg_tab_sr.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../resources/TurnFix.qrc"/>
 </resources>
 <connections/>
</ui>
